crates/humanode-runtime/src/lib.rs

```diff
@@ -393,11 +397,15 @@ parameter_types! {
     pub const TreasuryPotPalletId: PalletId = PalletId(*b"hmnd/tr1");
     pub const FeesPotPalletId: PalletId = PalletId(*b"hmnd/fe1");
     pub const TokenClaimsPotPalletId: PalletId = PalletId(*b"hmnd/tc1");
+    pub const CurrencySwapBalancesPotPalletId: PalletId = PalletId(*b"hmnd/sb1");
+    pub const CurrencySwapEvmBalancesPotPalletId: PalletId = PalletId(*b"hmnd/se2");
```

Comment:
Rename these accordingly after the pot pallet renames; also I suggest using the following id:

- `hmcsb/ne1` - humanode currency swap bridge native to evm
- `hmcsb/en1` - humanode currency swap bridge evm to native

Reply:
https://github.com/humanode-network/humanode/pull/673/commits/9afb2f649acbaad15eb4b98604ef24c015fa1260

Reply:
I really want there to be a numeric at the end, so let's drop the `b` and go with `hmcs/ne1` and `hmcs/en1`

Reply:
Updated - https://github.com/humanode-network/humanode/pull/673/commits/47e31fbf08ecc13f454ae2ab279c4944711d9db4